### PR TITLE
Readme gallery db schema change

### PR DESCRIPTION
**Pull request by lyndaidaii**
Summary of the changes

* add one column of enum EmbeddedReadmeFileType to package table

Addresses https://github.com/NuGet/NuGetGallery/issues/123

### CODE REVIEW

**Comment by joelverhagen** [comment]
Make sure to base this off `dev` in this PR. You can click the "Edit" button to change the base branch.
image

File: src/NuGet.Services.Entities/EmbeddedReadmeFileType.cs [hunk 1]
```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
```
Comment (zhhyu) [inline src/NuGet.Services.Entities/EmbeddedReadmeFileType.cs:3]: Leave one line here.

**Comment by lyndaidaii** [comment]
> Make sure to base this off `dev` in this PR. You can click the "Edit" button to change the base branch.
> image

Just realized after creating PR. Changed to base branch to dev

**Comment by zhhyu** [comment]
It lacks the change on the Package entity. Is it?

**Comment by lyndaidaii** [comment]
It was wrong branch. Close it. open new PR: https://github.com/NuGet/NuGetGallery/pull/8104

**Pull request closed without merging**